### ISSUE TITLE
Can't sync audio

### ISSUE THREAD

**Issue by Uatschitchun** [jellyfin/jellyfin-kodi#12]
Whenever I try to add my Audio-library to kodi I get an error message:

> There was an error, you can restart kodi tp progress the scanning (message is in german)

The kodi.log shows:
```
2019-03-02 19:33:25.692 T:140312462739200  NOTICE: JELLYFIN.entrypoint.default -> path: ?mode=addlibs params: {
                                                "mode": "addlibs"
                                            }
2019-03-02 19:33:25.692 T:140312462739200  NOTICE: JELLYFIN.default -> ---<[ default ]
2019-03-02 19:33:28.714 T:140312113293056  NOTICE: JELLYFIN.full_sync -> -->[ fullsync ]
2019-03-02 19:33:28.741 T:140312113293056  NOTICE: JELLYFIN.full_sync -> starting sync with [u'16795e3accb36058bd2737dc153fc696', u'Boxsets:', u'Boxsets:Refresh']
2019-03-02 19:33:28.795 T:140312113293056  NOTICE: JELLYFIN.helper.wrapper -> Processing Hörbücher: 16795e3accb36058bd2737dc153fc696
2019-03-02 19:33:29.011 T:140312113293056  NOTICE: JELLYFIN.objects.music -> UPDATE artist [2] (T)raumschiff Surprise Periode: abfcd8d0e584252b4889713f35cef20b
2019-03-02 19:33:29.210 T:140312113293056  NOTICE: JELLYFIN.objects.music -> UPDATE album [1] Loom - Das Hörspiel: 8fd29d90d88aec910face816ac5d8b7b
2019-03-02 19:33:29.239 T:140312113293056  NOTICE: JELLYFIN.database -> ERROR:: type: <class 'sqlite3.OperationalError'> value: no such column: strArtists
2019-03-02 19:33:29.240 T:140312113293056  NOTICE: JELLYFIN.database -> [jellyfin] 39 rows updated.
2019-03-02 19:33:29.258 T:140312113293056  NOTICE: JELLYFIN.database -> ERROR:: type: <class 'sqlite3.OperationalError'> value: no such column: strArtists
2019-03-02 19:33:29.259 T:140312113293056  NOTICE: JELLYFIN.database -> [music] 184 rows updated.
2019-03-02 19:34:17.796 T:140312113293056  NOTICE: JELLYFIN.full_sync -> ERROR:: full sync exited unexpectedly
2019-03-02 19:34:17.804 T:140312113293056  NOTICE: JELLYFIN.full_sync -> --<[ fullsync ]
2019-03-02 19:34:17.805 T:140312113293056  NOTICE: JELLYFIN.library -> ERROR:: no such column: strArtists
                                            Traceback (most recent call last):
                                              File "/home/uatschitchun/.kodi/addons/plugin.video.jellyfin/resources/lib/library.py", line 524, in add_library
                                                sync.libraries(library_id, update)
                                              File "/home/uatschitchun/.kodi/addons/plugin.video.jellyfin/resources/lib/full_sync.py", line 103, in libraries
                                                self.start()
                                              File "/home/uatschitchun/.kodi/addons/plugin.video.jellyfin/resources/lib/full_sync.py", line 193, in start
                                                self.process_library(library)
                                              File "/home/uatschitchun/.kodi/addons/plugin.video.jellyfin/resources/lib/full_sync.py", line 242, in process_library
                                                media[library['CollectionType']](library)
                                              File "/home/uatschitchun/.kodi/addons/plugin.video.jellyfin/resources/lib/helper/wrapper.py", line 38, in wrapper
                                                result = func(self, dialog=dialog, *args, **kwargs)
                                              File "/home/uatschitchun/.kodi/addons/plugin.video.jellyfin/resources/lib/full_sync.py", line 418, in music
                                                obj.album(album)
                                              File "/home/uatschitchun/.kodi/addons/plugin.video.jellyfin/resources/lib/helper/wrapper.py", line 97, in wrapper
                                                return func(*args, **kwargs)
                                              File "/home/uatschitchun/.kodi/addons/plugin.video.jellyfin/resources/lib/helper/wrapper.py", line 110, in wrapper
                                                return func(self, item, e_item=e_item, *args, **kwargs)
                                              File "/home/uatschitchun/.kodi/userdata/addon_data/plugin.video.jellyfin/jellyfin/objects/music.py", line 165, in album
                                                self.update_album(*values(obj, QU.update_album_obj))
                                              File "/home/uatschitchun/.kodi/userdata/addon_data/plugin.video.jellyfin/jellyfin/objects/kodi/music.py", line 154, in update_album
                                                self.cursor.execute(QU.update_album, args)
                                            OperationalError: no such column: strArtists
2019-03-02 19:34:19.294 T:140312487917312  NOTICE: JELLYFIN.entrypoint.default -> path: ?mode=managelibs params: {
                                                "mode": "managelibs"
                                            }
2019-03-02 19:34:19.316 T:140312487917312  NOTICE: JELLYFIN.default -> ---<[ default ]
```

What to do?

**Comment by flip-bs** [comment]
Same issue with a large music library more than 1K albums, i remove the music library in the jellyfin-kodi settings for the moment.  i can't provide any logs sorry :/

**Comment by jmeacoe** [comment]
In Kodi 18 they changed the music.db (music72) and are now using strArtistDisp in the album table.
They have also removed a couple tables.  I am looking into it, but am not positive what all the fields are for as there isn't really a data dictionary available.

I have gotten to the point where I can load my full library, but am prompted to update my tags whenever I select any music in Kodi.

**Comment by macbean101** [comment]
I was looking to make the switch from emby to jellyfin but have run in to this issue.  So far, the only library i've set up is music so i ran in to this issue.  Is there a fix available for this?

**Comment by oddstr13** [comment]
@macbean101 should be fixed in #32, but I would like an additional test in Kodi 18 before merging.

You can test by installing https://github.com/jmeacoe/jellyfin-kodi/archive/master.zip via install plugin from zip in Kodi, please report back in #32 if you do test it.
